### ISSUE TITLE
Search go to first instance 

### ISSUE THREAD

**Issue by premell** [qutebrowser/qutebrowser#6450]
When searching, etc "/foo",  and hitting enter it takes you to the top of the page and you then have to click "n" to go to the first occurence of "foo". It should take you to the first occurence by default

**Comment by The-Compiler** [comment]
Works fine for me. Please provide the information asked for in the [issue template](https://github.com/qutebrowser/qutebrowser/issues/new?assignees=&labels=&template=1_Bug_report.md).

**Comment by premell** [comment]
**Version info**:
qutebrowser v2.2.1
Git commit: 
Backend: QtWebEngine 5.15.3, Chromium 87.0.4280.144
Qt: 5.15.2

CPython: 3.9.4
PyQt: 5.15.4


**Does the bug happen if you start with `--temp-basedir`?**:
No it works as intended with --temp-basedir


**Description**
searching for a word takes you to the top of the page


**How to reproduce**
Any site, I tested on archlinux and searched for "copyright" since that only appears on the bottom

**Comment by The-Compiler** [comment]
Could you show your `:config-diff`? Does it work with your usual setup but without the custom config? If so, can you try finding out what in the config is causing this?

**Comment by premell** [comment]
So I removed the "content.noficiations.enabled" from my old config and the problem disapeared. However when I tried adding  back I was unable to reproduce the bug. Can it be that my config was created before you made some kind of an update and that caused the problem? 

aliases = 
auto_save.interval = 0
auto_save.session = false
bindings.commands = {"normal": {"J": "tab-prev", "K": "tab-next", "gJ": "tab-move -", "gK": "tab-move +"}}
colors.completion.category.bg = #282a36
colors.completion.category.border.bottom = #282a36
colors.completion.category.border.top = #282a36
colors.completion.category.fg = #f8f8f2
colors.completion.even.bg = #282a36
colors.completion.fg = #f8f8f2
colors.completion.item.selected.bg = #44475a
colors.completion.item.selected.border.bottom = #44475a
colors.completion.item.selected.border.top = #44475a
colors.completion.item.selected.fg = #f8f8f2
colors.completion.match.fg = #ffb86c
colors.completion.odd.bg = #282a36
colors.completion.scrollbar.bg = #282a36
colors.completion.scrollbar.fg = #f8f8f2
colors.downloads.bar.bg = #282a36
colors.downloads.error.bg = #282a36
colors.downloads.error.fg = #ff5555
colors.downloads.stop.bg = #282a36
colors.downloads.system.bg = none
colors.hints.bg = #282a36
colors.hints.fg = #bd93f9
colors.hints.match.fg = #e0e0e0
colors.keyhint.bg = #282a36
colors.keyhint.fg = #bd93f9
colors.keyhint.suffix.fg = #44475a
colors.messages.error.bg = #282a36
colors.messages.error.border = #282a36
colors.messages.error.fg = #ff5555
colors.messages.info.bg = #282a36
colors.messages.info.border = #282a36
colors.messages.info.fg = #6272a4
colors.messages.warning.bg = #282a36
colors.messages.warning.border = #282a36
colors.messages.warning.fg = #ff5555
colors.prompts.bg = #282a36
colors.prompts.border = 1px solid #282a36
colors.prompts.fg = #8be9fd
colors.prompts.selected.bg = #44475a
colors.statusbar.caret.bg = #282a36
colors.statusbar.caret.fg = #ffb86c
colors.statusbar.caret.selection.bg = #282a36
colors.statusbar.caret.selection.fg = #ffb86c
colors.statusbar.command.bg = #282a36
colors.statusbar.command.fg = #ff79c6
colors.statusbar.command.private.bg = #282a36
colors.statusbar.command.private.fg = #e0e0e0
colors.statusbar.insert.bg = #181920
colors.statusbar.insert.fg = #ffffff
colors.statusbar.normal.bg = #282a36
colors.statusbar.normal.fg = #f8f8f2
colors.statusbar.passthrough.bg = #282a36
colors.statusbar.passthrough.fg = #ffb86c
colors.statusbar.private.bg = #282a36
colors.statusbar.private.fg = #e0e0e0
colors.statusbar.progress.bg = #282a36
colors.statusbar.url.error.fg = #ff5555
colors.statusbar.url.fg = #f8f8f2
colors.statusbar.url.hover.fg = #8be9fd
colors.statusbar.url.success.http.fg = #50fa7b
colors.statusbar.url.success.https.fg = #50fa7b
colors.statusbar.url.warn.fg = #f1fa8c
colors.tabs.bar.bg = #44475a
colors.tabs.even.bg = #44475a
colors.tabs.even.fg = #f8f8f2
colors.tabs.indicator.error = #ff5555
colors.tabs.indicator.start = #ffb86c
colors.tabs.indicator.stop = #50fa7b
colors.tabs.indicator.system = none
colors.tabs.odd.bg = #44475a
colors.tabs.odd.fg = #f8f8f2
colors.tabs.selected.even.bg = #282a36
colors.tabs.selected.even.fg = #f8f8f2
colors.tabs.selected.odd.bg = #282a36
colors.tabs.selected.odd.fg = #f8f8f2
colors.webpage.darkmode.enabled = false
completion.height = 35%
completion.show = always
completion.use_best_match = true
content.blocking.adblock.lists = ["https://easylist.to/easylist/easylist.txt", "https://easylist.to/easylist/easyprivacy.txt"]
content.blocking.method = both
https://reddit.com: content.notifications.enabled = false
hints.border = 1px solid #282a36
session.default_name = random
statusbar.padding = {"bottom": 6, "left": 8, "right": 8, "top": 6}
statusbar.show = in-mode
tabs.favicons.scale = 1
tabs.indicator.width = 1
tabs.padding = {"bottom": 6, "left": 8, "right": 8, "top": 6}
tabs.position = top
tabs.show = switching
tabs.show_switching_delay = 1500
url.default_page = ~/.config/qutebrowser/startpage/index.html
url.open_base_url = true
url.searchengines = {"DEFAULT": "https://google.com/search?q={}", "o": "https://odysee.com/$/search?q={}", "y": "https://www.youtube.com/results?search_query={}"}
url.start_pages = ~/.config/qutebrowser/startpage/index.html

**Comment by The-Compiler** [comment]
> Can it be that my config was created before you made some kind of an update and that caused the problem?

I don't think so - nothing in that area changed recently as far as I remember.

> However when I tried adding back I was unable to reproduce the bug. 

I'm going to close this then, as there's little I can do without more information - if you find a reliable way to reproduce this, let me know, though!

**Comment by premell** [comment]
qutebrowser v2.2.3
Git commit:
Backend: QtWebEngine 5.15.4, Chromium 87.0.4280.144
Qt: 5.15.2

CPython: 3.9.5
PyQt: 5.15.4

I think I found the problem,  `statusbar.show  global: in-mode` reproduces the problem

I tested with this page https://en.wikipedia.org/wiki/CD_Projekt and searched for "Categories"
